modules/services/wpaperd.nix

```diff
@@ -45,5 +45,24 @@ in {
         source = tomlFormat.generate "wpaperd-wallpaper" cfg.settings;
       };
     };
+
```

Comment:
Should make the config section start with

``` nix
  config = mkIf cfg.enable {
    assertions = [
      (lib.hm.assertions.assertPlatform "services.wpaperd" pkgs lib.platforms.linux)
    ];

    home.packages = [ cfg.package ];
```

Unfortunately GitHub doesn't allow me to put this comment in the right place.